internal/cmd/operator-sdk/cli/cli.go

```diff
@@ -90,9 +91,7 @@ func GetPluginsCLIAndRoot() (*cli.CLI, *cobra.Command) {
 	)
 	gov4AlphaBundle, _ := plugin.NewBundle(golang.DefaultNameQualifier, plugin.Version{Number: 4, Stage: stage.Alpha},
```

Comment:
Do we want to keep this as alpha? Should we change this to:
```suggestion
	gov4Bundle, _ := plugin.NewBundle(golang.DefaultNameQualifier, plugin.Version{Number: 4},
```
Or:
```suggestion
	gov4Bundle, _ := plugin.NewBundle(golang.DefaultNameQualifier, plugin.Version{Number: 4, Stage: stage.Stable},
```

Reply:
Okay to disregard this based on https://github.com/operator-framework/operator-sdk/pull/6352#discussion_r1131465089